cmd/gcp-controller-manager/csr_approver.go

```diff
@@ -54,6 +54,11 @@ const (
 	tpmKubeletUsername    = "kubelet-bootstrap"
 
 	authFlowLabelNone = "unknown"
+
+	signerAPIClientCert      = "kubernetes.io/kube-apiserver-client"
+	signerKubeletClientCert  = "kubernetes.io/kube-apiserver-client-kubelet"
+	signerKubeletServingCert = "kubernetes.io/kubelet-serving"
+	signerLegacyUnknown      = "kubernetes.io/legacy-unknown"
```

Comment:
use constants in vendor/k8s.io/api/certificates/v1beta1/types.go ?

Reply:
Done